### ISSUE TITLE
make 404 page

### ISSUE THREAD

**Issue by minsooerickim** [citrushack/cutiehack2022#5]
make a generic 404 page with the error code and message saying "The page you are looking for does not exist."

**Comment by Aspireve** [comment]
Hey @minsooerickim  could I try doing this?

**Comment by Aspireve** [comment]
Hey!! Is this website just a page which then leads to the Next.js website? Cause that's what's loading for me
![image](https://user-images.githubusercontent.com/93852415/179343234-ce2337fc-21f1-4f97-b524-d1aa57da3854.png)

**Comment by minsooerickim** [comment]
@Aspireve Hey! What you are seeing is the Next.js website; we will be building on that page you are seeing; I just added a welcome text so it's not just a blank screen. 

But yeah! You could totally work on this if you would like. We will have an intro meeting with all the web dev committee members sometime early next week and we'll go over the basics there too and a little bit of what code practices we would like you guys to follow! If you want, you can assign yourself this issue and start working on it after we have the intro meeting!

let me know if you have any other questions! :)

**Comment by Aspireve** [comment]
Oh!! That's great!! But I am actually quite a beginner in this field and don't know much about a lot of stuff, so would I still be able to help in this? [its a bit intimidating for me]

**Comment by minsooerickim** [comment]
> Oh!! That's great!! But I am actually quite a beginner in this field and don't know much about a lot of stuff, so would I still be able to help in this? [its a bit intimidating for me]

yes of course! If you haven't already, make sure to join our discord. We can talk more about it there

https://discord.gg/SC4jA2uj6W
